live-examples/js-examples/array-join.html

```diff
@@ -1,12 +1,13 @@
 <pre>
-<code id="static-js">// create an array
-var elements = ['Fire', 'Wind', 'Rain'];
+<code id="static-js">var array1 = ['Fire', 'Wind', 'Rain'];
```

Comment:
This naming seems inconsistent with the "beasts" name above. I think I like the "beasts" choice for indexOf. Not sure what is a good descriptive name for this array though.

Reply:
I thought it was referring to elements of nature - as in the classical Greek elements :)

Reply:
After our conversation yesterday, I'm happy making some of the naming content-descriptive. So we can use beasts and even elements. After all they are not very abstract.